debian/libibverbs1.symbols

```diff
@@ -1,6 +1,7 @@
 libibverbs.so.1 libibverbs1 #MINVER#
  IBVERBS_1.0@IBVERBS_1.0 1.1.6
  IBVERBS_1.1@IBVERBS_1.1 1.1.6
+ IBVERBS_1.3@IBVERBS_1.3 1.1.12
  ibv_ack_async_event@IBVERBS_1.0 1.1.6
```

Comment:
The last PR changed the package version string for libibverbs, this .symbols update was generated based on the old version string, so it needs to be redone.

Be careful here since Doug merged the stuff for release 13 now, ideally build the .symbols for the 12 official release and then another update for 13.